src/main/webapp/i18n/de/exam.json

```diff
@@ -47,7 +47,7 @@
             "examStudentReviewStart": "Beginn der Klausureinsicht",
             "examStudentReviewEnd": "Ende der Klausureinsicht",
             "duration": "Dauer",
-            "nrOfStudents": "Anzahl der registrierten Studenten",
+            "nrOfStudents": "Anzahl an Registrierungen",
```

Comment:
Maybe "Anzahl registrierter Studierenden"?

Reply:
solved in 51600a0